text/rocq-roadmap.md

```diff
@@ -5,7 +5,7 @@ Rocq Long Term Roadmap
 
 The Rocq Development Team, building on 40 years of experience with Coq, is pleased to announce the Rocq Prover, the next-generation proof assistant. The Rocq Prover is designed to empower a diverse range of users, from mathematicians seeking formal rigor to software engineers building high-assurance systems.
 
-While Rocq leverages the rich heritage of Coq, it embraces a more agile development philosophy. This allows us to deliver impactful changes more frequently, accelerating innovation and user experience improvements.
+Rocq builds upon the rich heritage and diverse ecosystem of Coq. We promote a development methodology which achieves the delicate balance of being mindful of compatibility and ecosystem maintainability, while sufficiently agile to deliver impactful changes frequently, boosting research and innovation, and continuously improving user experience.
 
 This roadmap outlines our vision for Rocq's future, focusing on four key pillars:
```

Comment:
At some point, would you split the long lines in this file?

Reply:
If you want this to happen, feel free to open a PR doing it. Personally, I don't quite see the benefit.

Reply:
For one thing, it lets you comment on smaller parts of the text.  I think of it as standard procedure, just like for OCaml source code--really long lines are not as easy to read/work with.  But to each his own.

Reply:
OK, then I will do it in a commit on master after this PR is merged (assuming there are no other PR open on this text at the same time, to avoid merge conflicts).

Reply:
If I can hazard a comment: I think that it's mostly a tooling question (do your tools nudge you towards hard-wrapping or do they nudge you towards long line?), where older tools typically favored hard-wrapping to make the work of some line-oriented tools easier, while newer tools typically favor long lines for text documents (unstructured, unindented), for example Markdown.

In Github Markdown, and most Markdown processors these days, you are punished for hard-wrapping because it keeps the newlines in the rendered output (while the original markdown specification is design to consume hard-wrapped text nicely, by requesting two spaces at the end to force a newline). git-centric workflows are trying to work better with non-wrapped texts these days, for example some diff viewers will show you a word diff by default instead of a line diff, making any issue with long lines go away. On the other hand, hard-wrapped texts tend to be fairly unstable to reflows, and so they don't work so nice with line-based diff tools either.

Personally I keep hard-wrapping in Emacs out of habit, but I un-wrap my text before sending it to any web form. I don't have a strong preference for the wrapping strategy to use on versioned markdown documents.

Reply:
> In Github Markdown, and most Markdown processors these days, you are punished for hard-wrapping because it keeps the newlines in the rendered output

I believe this is only the case for the markdown used in comments, but not for the markdown used in documents.